.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       - checkout
       - ruby/install-deps
       - cypress/install:
+          node-cache-version: v2 # Invalidate earlier caches to pick up latest Cypress version
```

Comment:
follow on from https://github.com/sul-dlss/happy-heron/pull/3674/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47

see `node-cache-version` description in docs: https://circleci.com/developer/orbs/orb/cypress-io/cypress#commands-install (thanks @mjgiarlo for the fix and the doc link)